router/pump.go

```diff
@@ -215,18 +215,18 @@ func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool, inactivityTim
 	p.update(event)
 	go func() {
 		for {
-			debug("pump.pumpLogs():", id, "started")
+			debug("pump.pumpLogs():", id, "started, tail:", getopt("TAIL", "all"))
```

Comment:
i'd rather see `getopt` called once. can you update this to set a variable use that in these two places?

Reply:
yes, fair point - updated